lib/inspec/runner.rb

```diff
@@ -201,9 +200,9 @@ def add_target(target, _opts = [])
 
     def supports_profile?(profile)
       unless profile.supports_runtime?
-        raise "This profile requires #{Inspec::Dist::PRODUCT_NAME} version "\
+        raise "This profile requires product version "\
```

Comment:
I think this phrasing will lead to confusion - when a user hits this error, they need to know specifically what component has the faulty version (workstation? knife? kitchen? audit cookbook? inspec/cinc?). It seems to me that the product name needs to remain a constant reference here. 

I'd like to understand why that is problematic for the cinc project, and look for alternatives.

Reply:
the runner is used in the core-gem, which is in use in other 3rd party products like kitchen-terraform. When the runner executes in that context, it outputs a Chef wordmark. Changing it for a generic word (I reformulated to "the insepc-core gem" for clarity) ensures compliance of 3rd party projects that use inspec-core as a library.

Reply:
Is there a way to detect if it's wrapped in the Chef binstub maybe?